main.go

```diff
@@ -268,9 +270,12 @@ func serve(logger log.Logger, addr string, writers []writer, readers []reader) e
 		duration := time.Since(begin).Seconds()
 		writeProcessingDuration.WithLabelValues(writers[0].Name()).Observe(duration)
 
+		level.Debug(logger).Log("msg", "/write", "duration", duration) //nolint:errcheck
```

Comment:
Similar comment here. This is already provided as a Prometheus metric, see line 271. Do we really need to provide it as log message and metric?

Reply:
This is fine because it is a debug log.